test/unit/tx.js

```diff
@@ -100,32 +106,32 @@ describe('Tx', function () {
   })
 
   it('classify: invalid hash', () => {
-    expect(() => classify('aaaaa')).to.throw('Not a valid hash')
+    expect(() => classify('aaaaa')).to.throw(InvalidHashError, 'Not a valid hash')
```

Comment:
where do we use `classify` internally? in another PR we renamed it because I thought it's only used to determine pointer keys or is this another function? see https://github.com/aeternity/aepp-sdk-js/pull/1334#discussion_r763200657

@davidyuk maybe you can also comment this 😅